perun-core/src/test/java/cz/metacentrum/perun/core/entry/ResourcesManagerEntryIntegrationTest.java

```diff
@@ -885,7 +905,7 @@ public void updateResource() throws Exception {
 		facility = setUpFacility();
 		resource = setUpResource();
 
-		Resource resourceToUpdate = resourcesManager.createResource(sess, resource, vo, facility);
```

Comment:
I do not understand why I need 2 resources in this test? I created one and modified it, then checked returned resource with getResourceById result, but I do not see a reason for second resource.